### PR TITLE
Feat/user resources

### PR DESCRIPTION
**Pull request by Cliftonz**
Here are some updates to the website to include a google forms till we can get something more persistent. I was trying to learn more and I added a resource section, CSGlabs page, and updated the descriptions of the officers.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - name: "Matthew Hartman"
   occupation: "President"
-  description: "#!/bin/description"
+  description: "Mathew is a computer Science and math major whos interests are an enigma. He started his interests in security with the Computer Security class here at IUS."
```
Comment (TheOtherUnknown) [inline _data/people.yml]:
```suggestion
  description: "Mathew is a Computer Science and math major whose interests are an enigma. He started his interests in security with the Computer Security class here at IUS."
```

File: _includes/aboutus.html [hunk 2]
```diff
@@ -5,5 +5,5 @@ <h2>ABOUT US</h2>
         <li>Provide its members with technical education and hands on experience while creating connections to the real-world industry</li>
         <li>Educate the public about security awareness</li>
         <li>Enhance the reputation of the university by supplying the industry with trained professionals in the field of Cyber Security and participating in national security competitions</li></ol>
-        <p>To get involved, attend one of our scheduled meetings as indicated on our <a href="#schedule">calendar</a>.</p>
+        <p>To get involved, attend one of our scheduled meetings as indicated on our <a href="#schedule">calendar</a> or use the <a href="#Interest Form">form</a> down below to get put on our mailing list.</p>
```
Comment (TheOtherUnknown) [inline _includes/aboutus.html]:
```suggestion
        <p>To get involved, attend one of our scheduled meetings as indicated on our <a href="#schedule">calendar</a> or use the <a href="#Interest Form">form</a> down below to join our mailing list.</p>
```

File: _layouts/home.html [hunk 3]
```diff
@@ -46,5 +54,43 @@ <h2>SCHEDULE</h2>
...
+             </section>       
+        </div>
+    </div>
+</section>
+<br><br>
 
+<section id="resources" class="index-about">
+    <div class="container about-body">
+        <h2>Resources</h2>
+        <p> If you want to read more into Computer Science and Security topics, then we recommend this websites:
```
Comment (TheOtherUnknown) [inline _layouts/home.html]: I would remove this sentence.

File: _layouts/home.html [hunk 4]
```diff
@@ -46,5 +54,43 @@ <h2>SCHEDULE</h2>
 </section>
 <br><br>
 
+<section id="Interest Form" class="index-schedule">
+    <div class="container schedule-body">
+        <div class="row">
+            <section class="col-sm-12">
+                <h2>For More information please fill out this form</h2>
```
Comment (TheOtherUnknown) [inline _layouts/home.html]:
```suggestion
                <h2>Mailing list sign-up</h2>
```

File: _layouts/home.html [hunk 5]
```diff
@@ -46,5 +54,43 @@ <h2>SCHEDULE</h2>
...
+                <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSe4G1Dl1mrnlTnKrIXhlsAmsvUzTmCNywHFqcVDVnXx3fH4Pg/viewform?embedded=true" width="640" height="1547" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```
Comment (TheOtherUnknown) [inline _layouts/home.html]: I don't much like the styles on the iframe. It is not mobile friendly at all.

Comment (Cliftonz) [reply _layouts/home.html]: I think I have fixed it. I put the items in containers in css form.

**Comment by TheOtherUnknown** [comment]
[Cleanup iframe styles for both mobile and desktop](https://trello.com/c/TcQRzgAu/34-cleanup-iframe-styles-for-both-mobile-and-desktop)

**Pull request merged**